apps/frontend/utils/aggregate/flood/floodDetailedAggregateKeys.ts

```diff
@@ -0,0 +1,133 @@
+import { FloodSpecific, KoboCommonKeys } from '@wfp-dmp/interfaces';
+
+export const groupKey = KoboCommonKeys.commune;
+
+export const firstKeys = [KoboCommonKeys.province, KoboCommonKeys.district];
+export const sumKeys = [
```

Comment:
Could you do a deconstruction instead if we're summing all but two columns?

{RicePrice, threat, ...otherCategoriesToSum} = FloodSpecific

Something like that?

Reply:
You could extract it as well and not use it, but you see my point. I think we could avoid long lists like that in a few places in the code.

Reply:
I refactored sumKeys for now. I want to think more about how to restructure FloodSpecifc and commonfields